core/backend/src/GeographicCRSServices.ts

```diff
@@ -0,0 +1,40 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+/** @packageDocumentation
+ * @module iModels
+ */
+
+import { Range2dProps } from "@itwin/core-geometry";
+import { IModelHost } from "./IModelHost";
+
+/** Describes a geographic coordinate reference system produced by [[getAvailableCoordinateReferenceSystems]].
+ * @beta
+ */
+export interface AvailableCoordinateReferenceSystemProps {
+  /** TODO: Document the fields. */
```

Comment:
Document each field of the available CRS. e.g., what can I do with the name - what APIs can I pass it to? What does "deprecated" mean and under what circumstances should I use or avoid using a deprecated CRS?

You will need to run `build` and then `extract-api` after you add documentation.

Reply:
Done. Running extract-api only changed slightly common/api/core-backend.api.md. I don't know if this is what was expected.

Reply:
Yep, as expected.